test/plugin/base_test.rb

```diff
@@ -1608,22 +1649,32 @@ def verify_json_log_entries(n, params, payload_type = 'textPayload')
         labels ||= request['labels']
         labels.merge!(entry['labels'] || {})
 
-        assert_equal \
-          "projects/#{params[:project_id]}/logs/#{params[:log_name]}", log_name
+        expected_log_name = \
+          if params[:log_name]
+            "projects/#{params[:project_id]}/logs/#{params[:log_name]}"
+          else
+            # For a request with multiple tags, each entry's log name should be
+            # associated to the entry index.
+            "projects/#{params[:project_id]}/logs/tag#{entries_count}"
+          end
+        assert_equal expected_log_name, log_name
         assert_equal params[:resource][:type], resource['type']
         check_labels resource['labels'], params[:resource][:labels]
         check_labels labels, params[:labels]
         if block_given?
-          yield(entry, i)
+          yield(entry, entries_count)
         elsif payload_type == 'textPayload'
           # Check the payload for textPayload, otherwise it's up to the caller.
-          verify_default_log_entry_text(entry['textPayload'], i, entry)
+          verify_default_log_entry_text(entry['textPayload'], entries_count,
+                                        entry)
         end
-        i += 1
-        assert i <= n, "Number of entries #{i} exceeds expected number #{n}"
+        entries_count += 1
+        assert entries_count <= n, "Number of entries #{entries_count}" \
```

Comment:
If you rename `entries_count` to `entry_count`, the error message will fit on the next line.

Reply:
Changed.